hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RDBStore.java

```diff
@@ -80,6 +82,13 @@ public RDBStore(File dbFile, ManagedDBOptions dbOptions,
     dbLocation = dbFile;
 
     try {
+      rocksDBCheckpointDiffer =
+          new RocksDBCheckpointDiffer(dbLocation.getAbsolutePath(), 1000,
```

Comment:
Make the _number of snapshots_ a configuration value.

Reply:
doen